README.md

```diff
@@ -98,6 +98,18 @@ emojiPopup.isShowing(); // Returns true when Popup is showing.
 The `rootView` is the rootView of your layout xml file which will be used for calculating the height of the keyboard.
 `emojiEditText` is the [`EmojiEditText`](emoji/src/main/java/com/vanniktech/emoji/EmojiEditText.java) that you declared in your layout xml file.
 
+The following code needs to be executed onStart() and after creating EmojiPopup instance:
+
+```java
+emojiPopup.start()
+```
+
+The following code needs to be executed in onStop():
+
+```java
+emojiPopup.stop()
+```
+
```

Comment:
Can we reuse the `OnAttachStateChangeListener` or add [`ActivityLifecycleCallbacks`](https://developer.android.com/reference/android/app/Application.ActivityLifecycleCallbacks) (we already have an `Activity` context in the popup) to do this on our own to make this easier to use?

Reply:
We might be able to :P Can you do it? :D

Reply:
I'll try 🙂